spec/jobs/trs_mentor_sync_job_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe TRSMentorSyncJob, type: :job do
+  let(:mentor_1) { create(:mentor, first_name: "Joe", last_name: "Bloggs") }
+  let(:mentor_2) { create(:mentor, first_name: "Agatha", last_name: "Christie") }
```

Comment:
Not a blocker, but if you added a bang, you wouldn't have to do the before block. 
```suggestion
  let!(:mentor_1) { create(:mentor, first_name: "Joe", last_name: "Bloggs") }
  let!(:mentor_2) { create(:mentor, first_name: "Agatha", last_name: "Christie") }
```

Reply:
resolved in [7537d17](https://github.com/DFE-Digital/itt-mentor-services/pull/302/commits/7537d17dc7d8cfa052c688cc1a228c0b8832e82f)